Gemfile.lock

```diff
@@ -190,7 +190,7 @@ GEM
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    typhoeus (1.3.1)
+    typhoeus (1.4.0)
```

Comment:
“Typhon, also Typhoeus, was a monstrous serpentine giant and one of the deadliest creatures in Greek mythology.”